src/types/onyx/Policy.ts

```diff
@@ -19,7 +19,7 @@ type Policy = {
     owner: string;
 
     /** The accountID of the policy owner */
-    ownerAccountID: number;
+    ownerAccountID?: number;
```

Comment:
Are all those `?:` markers added because the corporate policies do not support these properties at all?

Shouldn't we model a [discriminated union](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#discriminated-unions) instead?